.gitignore

```diff
@@ -1 +1,5 @@
 .DS_Store
+/package-lock.json
+/package.json
+/node_modules
+.gitignore
```

Comment:
Lol. Normally you shouldn't do this. Because the gitignore is like a global configuration that all team members need for the project. :)